### PR TITLE
Feat: Added FormTitle component

### PR DESCRIPTION
**Pull request by robinpiets**
### Added FormTitle component (see [design system](https://designsystem.amsterdam.nl/7awj1hc9f/p/537bb7-title-component))

### CODE REVIEW

**Comment by jonkoops** [comment]
I've created a Trello ticket for the Design System meeting so we can discuss the remaining points: https://trello.com/c/3JlryDPs/328-react

File: packages/asc-ui/src/components/FormTitle/FormTitle.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      >
+        {storyFn()}
+      </div>
+    ),
+  ],
+
+  parameters: {
+    notes: {
+      markdown:
+        '#### See component in <a href="https://designsystem.amsterdam.nl/7awj1hc9f/p/537bb7-title-component" target="_blank">Design system</a>',
```
Comment (jonkoops) [inline packages/asc-ui/src/components/FormTitle/FormTitle.stories.tsx:28]: Can we use the same style here as in the other components? So a seperate README.md file with the component status as 'Experimental'. See: https://github.com/Amsterdam/amsterdam-styled-components/blob/master/packages/asc-ui/src/components/Accordion/README.md

File: packages/asc-ui/src/components/FormTitle/FormTitle.test.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { render } from '@testing-library/react'
+import FormTitle from './FormTitle'
+
+describe('FormTitle', () => {
+  it('should render', () => {
+    const { container, queryByText } = render(<FormTitle>Foo</FormTitle>)
```
Comment (jonkoops) [inline packages/asc-ui/src/components/FormTitle/FormTitle.test.tsx:7]: Should this not be wrapped in a `ThemeProvider`?

File: packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+import { themeColor, themeSpacing } from '../../utils/themeUtils'
+
+export default styled.h1`
+  max-width: 620px;
+  margin: ${themeSpacing(6, 0, 8)};
+  padding-bottom: 6px;
```
Comment (jonkoops) [inline packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts:7]: We have to assume the designers were using a 5 point grid here, so this should be 1 `themeSpacing` unit on our code.

File: packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  border-bottom: 1px solid ${themeColor('tint', 'level5')};
+  color: ${themeColor('tint', 'level5')};
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 18px;
+  -webkit-font-smoothing: auto;
```
Comment (jonkoops) [inline packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts:13]: Why reset the `-webkit-font-smoothing` here?

File: packages/asc-ui/src/components/FormTitle/FormTitle.stories.tsx (hunk 1 above)
Comment (robinpiets) [reply packages/asc-ui/src/components/FormTitle/FormTitle.stories.tsx:28]: Sure!

File: packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts (hunk 4 above)
Comment (robinpiets) [reply packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts:13]: In `GlobalStyle.ts` the style rule `-webkit-font-smoothing: antialiased;` is set on the `body`. This makes the `font-weight: 700` does not look strong enough.

`-webkit-font-smoothing: antialiased;`
image

`-webkit-font-smoothing: auto;` (overwrite)
image

That's why I overwrite `antialiased`

File: packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts (hunk 3 above)
Comment (robinpiets) [reply packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts:7]: fixed

File: packages/asc-ui/src/components/FormTitle/FormTitle.test.tsx (hunk 2 above)
Comment (robinpiets) [reply packages/asc-ui/src/components/FormTitle/FormTitle.test.tsx:7]: yep

Comment (jonkoops) [reply packages/asc-ui/src/components/FormTitle/FormTitle.test.tsx:7]: Looks like this is still missing, can you add it as well?

File: packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts (hunk 3 above)
Comment (jonkoops) [reply packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts]: :ok:

File: packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts (hunk 4 above)
Comment (jonkoops) [reply packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts:13]: If this is a problem, let's remove the global one instead. I'd rather have font rendering consistent where possible.

Comment (robinpiets) [reply packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts:13]: I've asked @royderks and @chrisvanmook in the #frontend channel

Comment (robinpiets) [reply packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts:13]: Removed it

**Pull request merged**